src/test/rpc/AccountSet_test.cpp

```diff
@@ -300,6 +402,7 @@ class AccountSet_test : public beast::unit_test::suite
         testSetAndResetAccountTxnID();
         testSetNoFreeze();
         testDomain();
+        testGateway();
```

Comment:
Two of your gateway tests are failing.  That's cool, I very much want the increased test coverage, and I know you're working on it.  But I also don't want to annoy you by pointing out things I'm sure you already know.  And I'd also like to conserve my code review eyes for when you are ready for me to look at stuff.

So could you specifically let us know when you're ready for more code review?  That way you can push to your heart's content until you are ready.  Thanks.